src/api.js

```diff
@@ -119,6 +126,7 @@ const Api = (options = {}) => {
   const preflightCheck = get(options, 'preflightCheck', () => false);
   const preflightHandlers = {};
   const preRequestHook = get(options, 'preRequestHook');
+  const pruneResponse = get(options, 'pruneResponse', true);
```

Comment:
Naming?

Reply:
I'd put this option on the fields parameter itself (and default to true)